specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/FormRecognizer.json

```diff
@@ -2046,6 +2050,33 @@
       "minimum": 0,
       "maximum": 1,
       "x-nullable": false
+    },
+    "Appearance": {
+      "description": "Information about style of extracted text.",
+      "type": "object",
+      "required": [
+        "style",
+        "styleConfidence"
+      ],
+      "properties": {
+        "style": {
+          "description": "Extracted text can have one of following listed styles.",
```

Comment:
"Style of the extracted text: handwriting or printed."

Reply:
Fixed.